azure/services/securitygroups/securitygroups_test.go

```diff
@@ -307,6 +307,13 @@ func TestDeleteSecurityGroups(t *testing.T) {
 				m.Delete(gomockinternal.AContext(), "my-rg", "nsg-two")
 			},
 		},
+		{
+			name: "skipping network security group delete in custom VNet mode",
+			expect: func(s *mock_securitygroups.MockNSGScopeMockRecorder, m *mock_securitygroups.MockclientMockRecorder) {
+				s.IsVnetManaged().Return(false)
```

Comment:
you need to add this expect call to the other test cases with a return value of true

`s.IsVnetManaged().Return(true)`

Reply:
line 285 and 304

Reply:
Added, thanks